.envrc

```diff
@@ -0,0 +1,11 @@
+export GOBUILDDIR=$(pwd)/.gobuild
```

Comment:
This file is added to allow building the source in any folder as long as you use direnv.